cpp/src/umap/fuzzy_simpl_set/naive.cuh

```diff
@@ -315,6 +315,7 @@ void launcher(nnz_t n,
 
   raft::sparse::COO<value_t> in(stream, n * n_neighbors, n, n);
 
+  /*
   // check for logging in order to avoid the potentially costly `arr2Str` call!
```

Comment:
Can you open an issue to track the failure and link it here?

Reply:
Sure, here it is : https://github.com/rapidsai/cuml/issues/6339